grid.v3/mailbox.go

```diff
@@ -77,6 +80,10 @@ func newMailbox(s *Server, nsName string, size int) (*Mailbox, error) {
 	timeout, cancel := context.WithTimeout(context.Background(), s.cfg.Timeout)
 	err := s.registry.Register(timeout, nsName)
 	cancel()
+	if err != nil && strings.Contains(err.Error(), "etcdserver: requested lease not found") {
```

Comment:
Maybe add a comment that this error can be found at : https://github.com/coreos/etcd/blob/54928f5debb3b81633dd4fbe5e7d02c0d170dbf9/etcdserver/api/v3rpc/rpctypes/error.go#L34